loleaflet/src/layer/tile/TileLayer.js

```diff
@@ -2061,16 +2061,16 @@ L.TileLayer = L.GridLayer.extend({
 			console.log('Applying a delta of length ' + delta.length + ' pix size: ' + pixSize + '\nhex: ' + hex2string(delta));
 
 			// Green-tinge the old-Data ...
-//			for (var i = 0; i < pixSize; ++i)
-//			{
-//				oldData[i*4 + 1] = 128;
-//			}
+			//			for (var i = 0; i < pixSize; ++i)
```

Comment:
This doesn't look right. Same also applies to the following three lines.

Please see the changes of loleaflet/src/layer/tile/CanvasTileLayer.js on f9dfe72f889d2182e2da2649154f4de00a860afb